README.md

```diff
@@ -36,11 +36,18 @@ Every member of the product team is expected to work hard, be considerate of the
 
-Furthermore, any behavior or language which is unwelcoming—whether or not it rises to the level of harassment—is also strongly discouraged. Much exclusionary behavior takes the form of [microaggressions](http://www.vox.com/2015/2/16/8031073/what-are-microaggressions)—subtle put-downs which may be unconsciously delivered. Regardless of intent, microaggressions can have a significant negative impact on victims and have no place on our team.
+Any behavior or language that is unwelcoming is strongly discouraged. Such exclusionary behavior can take many forms, including [microaggressions](http://www.vox.com/2015/2/16/8031073/what-are-microaggressions)—subtle put-downs, whether deliberate or unconscious. Regardless of intent, microaggressions can have a significant negative impact on victims and have no place on our team.
 
-There are a host of behaviors and language common on tech teams which are worth noting as specifically unwelcome: Avoid “_well, actuallys_”—pedantic corrections that are often insulting and unproductive; make an effort not to interrupt your colleagues while they are speaking; never respond with surprise when someone asks for help; and take care neither to patronize your colleagues nor assume complete knowledge of a topic. This last point is especially important when talking about technical topics: Many women and people of color in the tech industry have many tales of being either [mansplained](https://www.guernicamag.com/daily/rebecca-solnit-men-explain-things-to-me/) about a field in which they are experts, or else excluded from learning opportunities because a colleague wouldn’t make an effort to answer questions—don’t be that person. Remember that your colleagues may have expertise you are unaware of, and listen at least as much as you speak.
+There are a host of behaviors and language common on tech teams which are worth noting as specifically unwelcome:
+
+- avoid “_well, actuallys_”—pedantic corrections that are often insulting and unproductive;
+- make an effort not to interrupt your colleagues while they are speaking;
+- never respond with surprise when someone asks for help;
+- and take care neither to patronize your colleagues nor assume complete knowledge of a topic.
+
+This last point is especially important when talking about technical topics: Women and people of color in the tech industry have many tales of being either [mansplained](https://www.guernicamag.com/daily/rebecca-solnit-men-explain-things-to-me/) about a field in which they are experts, or else excluded from learning opportunities because a colleague wouldn’t make an effort to answer questions. Don’t be that person. Remember that your colleagues may have expertise you are unaware of, and listen at least as much as you speak.
```

Comment:
@gfogle you make an important point here. Yes, as @danmactough articulates, the term "mansplaining" is both imbued with meaning but also pejorative. I agree that we can do better, and offer a clearer definition of the behavior we want to avoid. "Mansplaining" does violate the CoC, as both of you allude, because it suggests that only men exhibit this behavior. Meanwhile the core of intersectionality is the acknowledgement that oppressed groups can be themselves oppressive, and everyone has a responsibility to avoid these behaviors and strive to treat everyone else with the utmost respect. We're all in this together. 

That said, while Vox's CoC has been such a valuable jumping off point, I think we should scrap this paragraph entirely. How about adding a few more items to the list? Mainly:

```
- don't presume to explain a topic without knowing whether someone wants or needs an explanation
- strive to be inclusive in learning opportunities and technical conversations
- listen more than you speak
```